source/partials/_team_section.haml

```diff
@@ -22,7 +22,7 @@
   .container
     %ul.team-list
       - data.team.team_members.select{|k,v| !v['retired'] }.each do |team_member|
-        %li
+        %li.member
```

Comment:
Why this additional class when the parent has a identifying class like team-list?